src/server/adapters/logger/logProvider.ts

```diff
@@ -39,8 +45,8 @@ const getLogDNA = ({ apiKey, environment }: LogDNAParams): LogProvider => {
         }
     );
     return {
-        info: (message, { meta }): void => logger.info(message, { meta }),
-        error: (message, { meta }): void => logger.error(message, { meta }),
+        info: (message, { meta }): void => logger.info && logger.info(message, { meta }),
```

Comment:
Is it possible to use the optional chain feature? 
```suggestion
        info: (message, { meta }): void => logger.info?.(message, { meta }),
```

Reply:
sure